client/src/pages/CreditCard.js

```diff
@@ -0,0 +1,49 @@
+import React from "react";
+import { CardElement, injectStripe } from 'react-stripe-elements';
+
+class CreditCard extends React.Component {
+
+    handleSubmit = async (ev) => {
+
+        ev.preventDefault();
+
+        const cardElement = this.props.elements.getElement('card');
```

Comment:
i'm guessing this element you are trying to get is `<CardElement>` one on this page? and the reason for this is to send it to stripe's `confirmCardSetup` ya? 

if so - it's better to assign that element with an `id` that is unique to this page, and grab that element that way :)

Reply:
Yeah, I kind of just accepted this as some hand waving magic by the react-stripe-elements package.  I just tried adding an id directly to the <CardElement> I am rendering and then getting that element using document.getElementybyID() (not sure if this is the right way).  Unfortunately, this doesn't yield the same result as using this.props.elements.getElement(), which returns a special object specific to react-stripe-elements with lots of added properties.  Is there some other way I should go about it?  The react-stripe-elements documentation says to use this.props.elements.getElement('card') which will find the right card element since there can only be one.